opengrok-indexer/src/main/java/org/opengrok/indexer/util/CtagsUtil.java

```diff
@@ -54,6 +60,49 @@ public static boolean validate(String ctagsBinary) {
         return true;
     }
 
-    private CtagsUtil() {
+    /**
+     * Gets the base set of languages by executing {@code --list-languages} for
+     * the specified binary.
+     * @return {@code null} on failure to run, or a defined list
+     */
+    public static List<String> getLanguages(String ctagsBinary) {
+        Executor executor = new Executor(new String[]{ctagsBinary, "--list-languages"});
+        int rc = executor.exec(false);
+        String output = executor.getOutputString();
+        if (output == null || rc != 0) {
+            LOGGER.log(Level.WARNING, "Failed to get Ctags languages");
+            return null;
+        }
+
+        output = output.replaceAll("\\s+\\[disabled]", "");
+        String[] split = output.split("(?m)$");
+        List<String> result = new ArrayList<>();
+        for (String lang : split) {
+            lang = lang.trim();
+            if (lang.length() > 0) {
+                result.add(lang);
+            }
+        }
+        return result;
+    }
+
+    /**
+     * Creates a new instance, and attempts to configure it from the
+     * environment.
```

Comment:
should probably explain how it differs to `Ctags()` constructor and when/where to use it.